it/helpers/ComponentSpecBase.scala

```diff
@@ -20,12 +20,14 @@ import binders.FinancialTransactionsBinders
 import com.github.tomakehurst.wiremock.stubbing.StubMapping
 import helpers.servicemocks.AuthStub
 import models.FinancialDataQueryParameters
+import org.scalatest.matchers.should.Matchers
 import org.scalatest._
 import org.scalatest.concurrent.{Eventually, IntegrationPatience, ScalaFutures}
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.ws.WSResponse
 import play.api.{Application, Environment, Mode}
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
 
 trait ComponentSpecBase extends TestSuite with CustomMatchers
   with GuiceOneServerPerSuite with ScalaFutures with IntegrationPatience with Matchers
```

Comment:
You already have `Matchers` import in scope as part of `CustomMatchers`, can you remove it here